task1/src/helpers/client_helper.py

```diff
@@ -108,6 +111,7 @@ def parse_text(self, text):
             (r'^cmd!%s$' % Commands.ROCK_PAPER_SCISSORS, lambda _: self._command_message(Commands.ROCK_PAPER_SCISSORS)),
             (r'^cmd!%s:(.+)$' % Commands.GAME_STEP, lambda regex: self._process_game_step(regex, text)),
             (r'^cmd!%s(.+):(.+)$' % Commands.PRIVATE_MESSAGE, lambda regex: self._process_private_message(regex, text)),
+            (r'^cmd!%s$' % Commands.SERVER_TIME, lambda regex: self._process_server_time_message()),
```

Comment:
if you don't use regex, just type lambda _: self.self._process_server_time_message()

Reply:
will fix